src/main/java/spring/boot/com/sbparser/service/ReviewService.java

```diff
@@ -0,0 +1,8 @@
+package spring.boot.com.sbparser.service;
+
+import java.util.List;
+import spring.boot.com.sbparser.model.Review;
+
+public interface ReviewService {
```

Comment:
let's create UserService and ProductService with `addAll()` method as well

and also implement saving all the entities to the DB in this PR